packages/web-pkg/src/components/AppBanner.vue

```diff
@@ -55,6 +55,9 @@ export default defineComponent({
     const store = useStore()
     const router = useRouter()
     const appBannerSettings = unref(store.getters.configuration.currentTheme.appBanner)
+    const useAppBanner = computed(
```

Comment:
```suggestion
    const isAppBannerAvailable = computed(
```

Not sure if such a convention is used in this repo, but I sometimes find it helpful to have a hint of the variable being a boolean in the name already - and  `use[...]` usually hints at a composable?

Reply:
Yep, fully agree on this